src/Task/PhpCsFixerV2.php

```diff
@@ -82,15 +89,23 @@ public function run(ContextInterface $context)
         }
 
         $arguments->addOptionalArgument('--using-cache=%s', $config['using_cache'] ? 'yes' : 'no');
-        $arguments->addOptionalArgument('--path-mode=%s', $config['path_mode']);
+        $arguments->addOptionalArgument('--path-mode=intersect', $config['can_intersect']);
```

Comment:
did you try this out? please add tests to cover this change, as it is not working.
```diff
-intersect
+intersection
```